### PR TITLE
control-center: Fix memory leak

### PR DESCRIPTION
**Pull request by rbuj**
Reported by clang:

  control-center.c:84:2: warning: Potential leak of memory pointed to by 'action'
          g_slist_free(key_list);
          ^~~~~~~~~~~~

### CODE REVIEW

File: shell/control-center.c [hunk 1]
```diff
@@ -70,6 +70,8 @@ static GSList* get_actions_list(void)
 
 		if ((action->item = load_desktop_item_from_unknown(temp[1])) == NULL)
 		{
+			g_free ((char*)action->name);
```
Comment (muktupavels) [inline shell/control-center.c]: Perhaps `name` should not be const in first place?

Comment (rbuj) [reply shell/control-center.c]: thanks for the tip

File: shell/control-center.c [hunk 2]
```diff
@@ -81,7 +83,7 @@ static GSList* get_actions_list(void)
 		g_free(entry);
 	}
 
-	g_slist_free(key_list);
+	g_slist_free (key_list);
```
Comment (muktupavels) [inline shell/control-center.c]: Looks like unrelated change...

Comment (rbuj) [reply shell/control-center.c]: I appreciate your comments
